common/util/file_util.cc

```diff
@@ -161,6 +167,62 @@ absl::Status GetContents(absl::string_view filename, std::string *content) {
   return absl::OkStatus();
 }
 
+absl::StatusOr<std::unique_ptr<MemBlock>> AttemptMemMapFile(
+    absl::string_view filename) {
+#ifndef _WIN32
+  class MemMapBlock final : public MemBlock {
```

Comment:
Move the class to the anonymous namespace?

Reply:
It already is - it is defined inside the `AttemptMemMapFile()` function itself.